cmd/ci-images-mirror/main.go

```diff
@@ -320,6 +338,69 @@ func main() {
 	logrus.Info("Process ended gracefully")
 }
 
+func validateConfig(config *quayiociimagesdistributor.CIImagesMirrorConfig, registryConfig string, ciOperatorconfigPath string) error {
```

Comment:
I would collect all the errors and then return instead of return right after the first error.